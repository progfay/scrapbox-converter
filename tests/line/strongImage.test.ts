/* global describe it expect */

import { parse } from '@progfay/scrapbox-parser'
import convert from '../../src'
import pretty from 'pretty'

describe('strongImage', () => {
  it('Simple strongImage', () => {
    const input = `Strong Image
[[https://gyazo.com/5593f3bbe109c38ebf07c16dd25dc4c4]]`
    const html = convert(parse(input), 'projectName')
    expect(pretty(html)).toMatchSnapshot()
  })
})
