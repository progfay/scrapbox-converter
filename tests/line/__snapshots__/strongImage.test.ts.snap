// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`strongImage Simple strongImage 1`] = `
"<article>
  <h1 class=\\"title\\">Strong&nbsp;Image</h1>
  <section>
    <p>
      <amp-img src=\\"https://gyazo.com/5593f3bbe109c38ebf07c16dd25dc4c4/thumb/1000\\" layout=\\"intrinsic\\" height=\\"720\\" width=\\"1280\\" alt=\\"image\\" class=\\"strong-image\\"></amp-img>
    </p>
  </section>
</article>"
`;
